src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -85,40 +85,46 @@ namespace AppInstaller::CLI::Workflow
 
         if (m_ensureAcceptance)
         {
-            context << Workflow::EnsureLicenseAcceptance(/* showPrompt */ true);
+            context << Workflow::EnsurePackageAgreementsAcceptance(/* showPrompt */ true);
         }
     }
 
-    void EnsureLicenseAcceptance::operator()(Execution::Context& context) const
+    void EnsurePackageAgreementsAcceptance::operator()(Execution::Context& context) const
     {
+        if (WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::SkipAgreementsAcceptance))

```

Comment:
SkipAgreementAcceptance is not a command line option, it's a flag set by ComContext, where for Com flow, we should skip agreement checks

Reply:
Perhaps we shouldn't call this skip agreements, but something like "User already agreed to agreements".

Reply:
@denelon RE naming.

Reply:
Naming things is hard. I'm not sure how one would "read" this code. I agree we don't want this to be perceived as a way to programmatically skip agreements. I think the intent is either to indicate that we don't need to prompt because the agreement had already been accepted by the user or by the IT Administrator when set by policy. 

AgreementPreviouslyAccepted ?